standard_template/standard/clause_0_front_material.adoc

```diff
@@ -1,6 +1,6 @@
 .Preface
 
-The specification was an outcome of a European Union project, "Maximizing the Exploitation of Linked Open Data in Enterprise and Science" (MELODIES), run from 2013 to 2016, and released under a Creative Commons 4.0 Licence.
+The specification was an outcome of a European Union project, "Maximizing the Exploitation of Linked Open Data in Enterprise and Science" (MELODIES), run from 2013 to 2016, and released under a Creative Commons 4.0 Licence by Reading University.
```

Comment:
```suggestion
The specification was an outcome of a European Union project, "Maximizing the Exploitation of Linked Open Data in Enterprise and Science" (MELODIES), run from 2013 to 2016, and released under a Creative Commons 4.0 Licence by the University of Reading.
```

Reply:
Can you press the button to commit this? You have just resolved this thread without making a change. Same for the name typo where Jon added a separate comment.